Docs/pt-BR/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contribuindo
```

Comment:
Could you rename the directory to lower-case `docs` rather than `Docs`

With git you have to do it like:

```
git mv Docs docs
```

Reply:
```
$ git mv Docs docs
Rename from 'Docs' to 'docs/Docs' failed. Should I try again? (y/n) y
Rename from 'Docs' to 'docs/Docs' failed. Should I try again? (y/n) y
``

Reply:
```
$ git mv -f -v Docs docs
Renaming Docs to docs/Docs
fatal: renaming 'Docs' failed: Invalid argument
```